### ISSUE TITLE
Dropzone help text doesn't meet AA colour contrast

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#967]
**Describe the bug**
Dropzone help text doesn't meet AA colour contrast. Current contrast ratio is 4.41 out of a required 4.5.
